data-migration/2.x-to-3.x/README.md

```diff
@@ -0,0 +1,81 @@
+# Updating data from 2.x to 3.x

```

Comment:
Can we link to this document in the `Unreleased` chapter of the [CHANGELOG](https://github.com/research-software-directory/research-software-directory/blob/master/CHANGELOG.md).

Reply:
Added link to new section _3.0.0 (Unreleased)_ in CHANGELOG

Reply:
Looks OK now